sources/us/ca/city_of_hayward.json

```diff
@@ -0,0 +1,38 @@
+{
+    "coverage": {
+        "geometry": {
+            "type": "Point",
+            "coordinates": [
+                -122.080833,
+                37.668889
+            ]
+        },
+        "country": "us",
+        "state": "ca",
+        "city": "Hayward"
+    },
+    "data": "http://maps.hayward-ca.gov/arcgis/rest/services/OpenData/COH_Addresses/FeatureServer/0",
+    "website": "http://opendata.hayward-ca.gov/datasets/5b2d03233e0344c7a291c3f126cfa407_0",
+    "attribution": "City of Hayward and Fairview. Licensed for Public Use",
+    "type": "ESRI",
+    "conform": {
+        "type": "geojson",
+        "number": [
```

Comment:
Our tests aren't liking these arrays with one element. Can you make them just strings?